### PR TITLE
EURid code breaking changes 17 June 2020: registrar lock service, multiyear transfers

### PR DESCRIPTION
**Pull request by paullojorgge**

### CODE REVIEW

File: t/Net/DRI/Protocol/EPP/Extensions/EURid.t [hunk 1]
```diff
@@ -33,11 +33,11 @@ my ($rc,$s,$d,$co,$toc,$cs,$h,$dh,@c);
 ########################################################################################################
 ## Process greetings to select namespace versions
 # This greeting was taken off release_notes_march_2019
```
Comment (thedarkwinter) [inline t/Net/DRI/Protocol/EPP/Extensions/EURid.t]: Can you remove/correct this note please? Its misleading now :)
(note more more below. No need to change comments on all domain examples as only namespace has changed, but it feels relevant in the greeting)

File: t/Net/DRI/Protocol/EPP/Extensions/EURid/Contact.t [hunk 2]
```diff
@@ -35,7 +35,7 @@ my ($rc,$s,$d,$co,$toc,$cs,$h,$dh,@c);
 ## Greeting Examples taken from EPP_Guidelines_2_1_09
```
Comment (thedarkwinter) [inline t/Net/DRI/Protocol/EPP/Extensions/EURid/Contact.t]: Likewise with this noted

File: t/Net/DRI/Protocol/EPP/Extensions/EURid.t (hunk 1 above)
Comment (paullojorgge) [reply t/Net/DRI/Protocol/EPP/Extensions/EURid.t]: Makes sense. Comments removed/updated.

**Comment by thedarkwinter** [comment]
Thanks @paullojorgge . I will merge this when the registry pushes the update

**Pull request merged**